force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -356,14 +501,21 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<Person> createMultiplePersons(List<String> names, Integer age) {
 
+        List<Person> persons = new List<Person>(); // Initialize an empty list to store Person objects
+
         // Loop through each name in the provided list of names
             // Create a new Person object for each name
             
-            // Add the new Person object to the list of Person objects
-        
+        for (String name : names) {
+        // Create a new Person object for each name with the specified age
+        Person person = new Person(name, age);
+
+        // Add the new Person object to the list of Person objects
+        persons.add(person);
+        }
         
         // Return the list of Person objects
-        return null; // Replace null with the variable you used to store the result
+        return persons; // Replace null with the variable you used to store the result
```

Comment:
In your loop you don't need that person variable. 

 for (String name : names) {
            persons.add(new Person(name, age));
        }